### ISSUE TITLE
false negative unused namespace when using fully qualified keyword

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#281]
```
echo '(ns f (:require [clojure.string :as s])) :s/foo'  | /usr/local/bin/clj-kondo --lint -
linting took 33ms, errors: 0, warnings: 0
```

cc @souenzzo

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/058fbd614c09ee5b6ba989c1117e3e2620151c79

**Comment by borkdude** [comment]
@souenzzo:

```
$ echo '(ns f (:require [clojure.string :as s])) :s/foo' | clj -Sdeps '{:deps {clj-kondo {:git/url "https://github.com/borkdude/clj-kondo" :sha "058fbd614c09ee5b6ba989c1117e3e2620151c79"}}}' -A:clj-kondo --lint -
<stdin>:1:18: warning: namespace clojure.string is required but never used
linting took 127ms, errors: 0, warnings: 1
```

**Comment by viebel** [comment]
@borkdude It seems to fail on latest master

```bash
echo '(ns f (:require [clojure.string :as s])) :s/foo' | clj -Sdeps '{:deps {clj-kondo {:git/url "https://github.com/borkdude/clj-kondo" :sha "085c7bcccfa3b4cb1b87a7ba39180666d37ddd2d"}}}' -A:clj-kondo --lint -

<stdin>:1:18: warning: namespace clojure.string is required but never used
linting took 135ms, errors: 0, warnings: 1
```

**Comment by borkdude** [comment]
@viebel You'll need to use `::s/foo` for the aliased keyword to need the namespace reference.
